index.html

```diff
@@ -0,0 +1,274 @@
+    <nav id="nav-container">
+      <div class="mobile-menu">
+        <span class="material-icons hamburger">menu</span>
+      </div>
+      <div class="desktop-menu">
+        <div class="head-menu">
+          <a class="desktop-logo" href="index.html">
+            <img class="logo" src="./images/logo.png" alt="logo">
+          </a>
+        </div>
+        <ul class="nav-list-container">
+          <li ><a class="about-li" href="./pages/about.html">About</a></li>
+          <li><a href="#information">Information</a></li>
+          <li><a href="#bands">Bands</a></li>
+          <li><a href="#partners-container">Partners</a></li>
+          <li><a href="">News</a></li>
+          <li class="box-item"><a class="box-li" href="">CC Campaign</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <section id="home">
+    <h1 class="home-h1">"Hello! New Music World"</h1>
+    <h2 class="home-h2">GREEN PARK <br> OUTDOOR CONCERT <br> 2022</h2>
+    <p class="concert-description">International concert that presents outstanding performances by the world's greatest artists is taking place in October, in Argentina.</p>
+    <p class="concert-date">2022.10.15(SAT)~16()SUN</p>
+    <p class="concert-place">@ Green Park, Salta - Argentina</p>
+  </section>
+
+  <section id="information">
+    <h2 class="information-h2">Information</h2>
+    <ul class="information-list">
+      <li>
+        <div class="info-icon">
+          <span class="material-icons">graphic_eq</span>
+          <h3>Singers and Bands</h3>
+        </div>
+        <p>Take a look of the world's greatest artists participating this year.</p>
+      </li>
+      <li>
+        <div class="info-icon">
+          <span class="material-icons">event</span>
+          <h3>Porgram by day</h3>
+        </div>
+        <p>We have organize two amazing days with full of activities. You can take a look here!</p>
+      </li>
+      <li>
+        <div class="info-icon">
+          <span class="material-icons">sensor_door</span>
+          <h3>Backstage</h3>
+        </div>
+        <p>Look at behind-the-scenes. Meet our event planners and take a look of their amazing ideas for this year</p>
+      </li>
+      <li>
+        <div class="info-icon">
+          <span class="material-icons">map</span>
+          <h3>Getting here</h3>
+        </div>
+        <p> Click here for directions to Green Park Outdoor Concert in Salta - Argentina.</p>
+      </li>
+      <li>
+        <div class="info-icon">
+          <span class="material-icons">accessibility_new</span>
+          <h3>Accesibility</h3>
+        </div>
+        <p>We make every effort to provide all of our patrons with an enjoyable experience.</p>
+      </li>
+    </ul>
+    <button class="join-btn" type="button">Join Outdoor Concert 2022</button>
+    <a class="see-more-information" href="">SEE MORE INFORMATION HERE</a>
+  </section>
+
+  <section id="bands">
+    <h2 class="bands-h2">Featured Musisians and Bands</h2>
+  </section>
```

Comment:
- Good job adding the singers for the band dynamically 👍🏿 
- [ ] Kindly add your own  first image before the singers image, this will make your design look similar to the Behance design🤗 



Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_22](https://user-images.githubusercontent.com/69638013/163291134-75195294-06ab-4e8a-815c-e31cf14269bc.png)  | ![Screenshot_21](https://user-images.githubusercontent.com/69638013/163291149-873656d8-9f88-4620-a2e5-f83c076aca6e.png)